src/framework/components/scroll-view/component.js

```diff
@@ -32,6 +33,8 @@ var _tempScrollValue = new Vec2();
  *
  * @property {number} bounceAmount Controls how far the content should move before bouncing back.
  * @property {number} friction Controls how freely the content should move if thrown, i.e. By flicking on a phone or by flinging the scroll wheel on a mouse. A value of 1 means that content will stop immediately; 0 means that content will continue moving forever (or until the bounds of the content are reached, depending on the scrollMode).
+ * @property {boolean} useMouseWheel Whether to use mouse wheel for scrolling (horizontally and vertically) when mouse is within bounds.
+ * @property {vec2} mouseWheelSensitivity Mouse wheel horizontal and vertical sensitivity. Only used if useMouseWheel is set.
```

Comment:
is this commonly exposed as two numbers, or just one that applies to both?

Reply:
I think support for 'X' direction was not widely used so most systems only use the y direction. I see no reason why to not add support for X-direction, and it would make sense to have separate controls. In case a developer wants the Y control, but not X, they can set the X sensitivity to 0, for instance. And if they wish to add sensitivity controls in their game's Settings, I believe it is best to have control over both if needed.